draft-ietf-taps-interface.md

```diff
@@ -379,13 +380,16 @@ Connection := Preconnection.Initiate()
 
 Connection -> Ready<>
 
+//---- Ready event handler begin ----
```

Comment:
Again, here we could use separate snippets of code to make it clearer that this is separate code, which gets called when the event happens.

Reply:
Suggested approach as above